src/main/scala/com/wavesplatform/history/HistoryWriterImpl.scala

```diff
@@ -78,7 +78,7 @@ class HistoryWriterImpl private(file: Option[File], val synchronizationToken: Re
 
         if (h % 100 == 0) db.compact(CompactFillRate, CompactMemorySize)
 
-        log.trace(s"Full Block(id=${block.uniqueId},txs_count=${block.transactionData.size}) persisted")
+        log.trace(s"Full Block $block(id=${block.uniqueId} persisted")
```

Comment:
I think, we don't need `block.uniqueId` here, because it is already logged (from Block.toString):
```scala
s"Block(${signerData.signature.trim}…"
```